test/statmodels.jl

```diff
@@ -30,3 +30,9 @@ x3   0.453058  0.72525 0.999172 0.5567
 @test sprint(show, StatsBase.PValue(NaN)) == "NaN"
 @test_throws ErrorException StatsBase.PValue(-0.1)
 @test_throws ErrorException StatsBase.PValue(1.1)
+
+try
+    throw(ConvergenceException(10))
+catch ex
+    @test sprint(Base.showerror, ex) == "failure to converge after 10 iterations"
```

Comment:
Hm, when I remove `Base`, I get this
```julia
julia> throw(ConvergenceException(10))
ERROR: StatsBase.ConvergenceException(10)
```
whereas with `Base`, I get what I expect
```julia
julia> throw(ConvergenceException(10))
ERROR: failure to converge after 10 iterations
```